### ISSUE TITLE
New translations files

### ISSUE THREAD

**Issue by ramoncodit** [GoteoFoundation/goteo#108]
Good evening @microstudi 
I have a quick question about .yml translations files.
In order to add new templates with its own label tags, I created new .yml file with its specific translations labels on path (changing <lang_id> by its language code and <new_file_name> by file name):
- Resources/translations/<lang_id>/<new_file_name>.yml

But seems that new translations file is not found. 
When I put all new file labels and translations on an existing translations .yml file, for example:
- Resources/translations/es/admin.yml
then worked.

I couldn't find anywhere to "include" my new .yml file, so my question is, what's the correct form to procede? Or in default, where can I find some related documentation?

Thanks in advance
Greetings

**Comment by microstudi** [comment]
Do you want to add these new labels in a plugin?

You must add the new group in your plugin (are you using  a custom plugin?)

Here is explained:
http://goteofoundation.github.io/goteo/docs/developers/extend.html#the-1

basically, with the function `addYamlTranslation`
```php
\Goteo\Application\Lang::addYamlTranslation('en', '{path-to-your-translations-dir}/en/some-file.yml');
```

**Comment by ramoncodit** [comment]
Hi @microstudi 
first of all, thanks for your quick response.

Responding at your question, no, I wasn't creating a new custom plugin.
I added a new table on database, a new route, controller, model and related template. So, in that new template, I added new labels to get translated and it didn't worked.

After I've took another look, I realised that where the translation are loaded:
- src/Goteo/Application/Config.php
It only needed to add new group in array:
```
    // Initial translation groups (groupped in yml files into Resources/translations/)
    static public $trans_groups = ['home', .... ];
```
Sorry in advance
Greetings
